projects/plugins/protect/src/js/components/admin-page/styles.module.scss

```diff
@@ -37,10 +37,10 @@
 .is-viewport-small {
 	// Add spacing between main and secondary in "sm" viewport size
 	.main {
-		margin-bottom: calc( var( --spacing-base ) * 3 );
+		margin-bottom: calc( var( --spacing-base ) * 3 ); // 24px
 	}
 }
 
 .connection-error-col {
-	margin-top: 25px;
+	margin-top: calc( var( --spacing-base ) * 3 + 1 ); // 25px
```

Comment:
I wasn't able to figure out an easy way to get this column to display, but my assumption is that applying an additional `px` value this way won't return the expected results as seen with the styling of the Modal component.

Reply:
Same issue as the Modal padding, added a `px` unit via [5f31b9b](https://github.com/Automattic/jetpack/pull/27175/commits/5f31b9b1c3cbd454730415f7adc4004a42e87adf).